pulsar/consumer_impl.go

```diff
@@ -72,6 +73,14 @@ func newConsumer(client *client, options ConsumerOptions) (Consumer, error) {
 		return nil, newError(SubscriptionNotFound, "subscription name is required for consumer")
 	}
 
+	if options.PriorityLevel <= 0 {
+		options.PriorityLevel = 0
+	}
+
+	if options.Type != Shared {
```

Comment:
Does we need this?

Reply:
Is PriorityLevel effectived when SubscriptionType is Shared?